android/assets/jsons/translations/template.properties

```diff
@@ -904,6 +906,15 @@ Unlocked at =
 Gain 2 free technologies = 
 All policies adopted = 
 
+# Religions
+
+Choose an Icon and name for your Religion = 
+Christianity = 
```

Comment:
These should definitely be auto generated. No need for that right now I think, but don't put this here manually at least